app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
```

Comment:
- [ ] Kindly make sure you modify the views in devise for registration and login to match the wireframes from the Sneak Peek and your styling as it is [required](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Modify%20the%20views%20in%20devise%20for%20registration%20and%20login%20to%20match%20the%20wireframes%20from%20the%20Sneak%20Peek%20and%20your%20styling.)

the form
![Capture](https://github.com/BenMKT/BlogApp/assets/57408419/00016314-8807-4af2-990b-7f058528efc5)

Your project
![Capture1](https://github.com/BenMKT/BlogApp/assets/57408419/fe3f1855-ed95-484d-91d6-df0ae6b85eba)
![Capture2](https://github.com/BenMKT/BlogApp/assets/57408419/f2565b46-6b68-4d47-83c2-6257cf52070e)




- [ ] Also I will suggest you add an `input` for `bio`  when we are signing up, to make sure we have data for bio
> | | signup | | the view | |
> |-|---------|-|--------|-|
> | |![Capture2](https://github.com/BenMKT/BlogApp/assets/57408419/d4fd52dc-486e-4bb5-ad4c-8fd1d1e4da52)| |![Capture3](https://github.com/BenMKT/BlogApp/assets/57408419/2d913659-b03b-46f8-bbcc-5329f77afcb7)| |

Reply:
- Modified the views in `devise` for registration and login to resolve bugs.